.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### Checklist for this issue
+If your issue is about the regex used in `email` rule, please note that as of version 1.12.0 this plugin is using the same regular expression that the [HTML5 specification suggests for browsers to use](https://html.spec.whatwg.org/multipage/forms.html#valid-e-mail-address). We will follow their lead and use the same check. If you think the specification is wrong, please report the issue to them. If you have different requirements, consider [using a custom method](http://jqueryvalidation.org/jQuery.validator.addMethod/).
```

Comment:
yeah thats what I meant initially :-).

we should then format this plain text to be human readable (and not using markdown markup).